src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1319,10 +1319,15 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                     }
 
                                     if ($ErrorView -eq 'ConciseView') {
+                                        $recommendedAction = $_.ErrorDetails.RecommendedAction
+                                        if (-not [String]::IsNullOrWhiteSpace($recommendedAction)) {
+                                            $recommendedAction = $newline + '  Recommendation: ' + $recommendedAction
+                                        }
+
                                         if ($err.PSMessageDetails) {
                                             $posmsg = ""${errorColor}${posmsg}""
                                         }
-                                        return $posmsg
+                                        return $posmsg + $recommendedAction
```

Comment:
should the recommendedAction be in a different color from the error message?

Reply:
I considered this but it would probably also require a new PS style parameter, but opted for the recommended action separator instead to keep people's visual Styles consistent